.github/workflows/ubuntu-noble.yml

```
@@ -69,15 +76,15 @@ jobs:
         timeout_minutes: 25
         max_attempts: 5
         command: |
-          apt-get -y update && apt-get upgrade -y &&
-          apt-get install -y \
-            libiberty-dev clang libomp-dev libopenmpi-dev libfabric-dev \
-            openmpi-bin ${{ matrix.compiler }} &&
-          for i in 8 9 10 11 12; do /opt/conda/envs/py3.${i}/bin/python -m pip install numpy perfetto dataclasses; done
+          apt-get update &&
+          apt-get upgrade -y &&
+          apt-get install -y autoconf bison build-essential clang environment-modules \
+            gettext libfabric-dev libiberty-dev libomp-dev libopenmpi-dev libtool m4 \
+            openmpi-bin python3-pip texinfo
```

```suggestion
          apt-get install -y libomp-dev libopenmpi-dev openmpi-bin
```
I suggest just installing the packages that are specific to tests. The other ones are already installed in the docker image. 
I know, the other workflows need cleaning up too.